src/scss/components/_header.scss

```diff
@@ -0,0 +1,248 @@
+$z-index--1010: 1010;
+$z-index--1009: 1009;
+
+$letter-spacing--0-2: -0.2px;
+
+$line-height--1-2: 1.2;
+
+$color-border-default: #dcdacb;
+$color-border-dark: #989586;
+$color-border-blue:#2896a9;
+
+$max-width--122: 122px;
+$min-width--240:240px;
+
+$padding--0--12: 0 12px;
```

Comment:
I think we don't need these line from 15 to 23.

Please define $spacing variable
$spacing-0: 0
$spacing-x: x

And create new file _padding.scss
.p-0 {
padding: 0
}
.pl-8{
padding-left: $spacing-8
}
.pr-8 {
padding-right: $spacing-8
}
.px-8{
@extends .pl-8
@extends .pr-8
}
.pt-8{
padding-top: $spacing-8
}
.pb-8{
padding-bottom: $spacing-8
}
.px-8{
@extends .pt-8
@extends .pb-8
}

Reply:
Do you resolve this case?